servers/partner-ci/common.yaml

```diff
@@ -402,6 +402,22 @@
                 echo "snapshot_id_number=${snapshots_id}"
                 python "${CONV_SNAP_PATH}"
 - builder:
+    name: get-mirror-id
+    builders:
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            [[ "${SNAPSHOTS_ID}" != *"released"* ]]
+          steps:
+            - shell: |
+                [[ -z "${SNAPSHOTS_ID}" ]] && { echo "SNAPSHOTS_ID is empty"; exit 1; }
+                wget --no-check-certificate -O ${SNAPSHOT_ARTIFACTS_FILE} \
+                ${SNAPSHOTS_URL/SNAPSHOTS_ID/$SNAPSHOTS_ID}
+                MOS_UBUNTU_MIRROR_ID=$(grep "^MOS_UBUNTU_MIRROR_ID" < snapshots.params | cut -d= -f2)
```

Comment:
On the product-ci we have a special file to use snapshot.params as variables:
for ex. `https://product-ci.infra.mirantis.net/view/9.x/job/9.x.snapshot/686/artifact/snapshots.sh`
if we use the special macros we could download the snapshots.sh file and execute it to export variables with IDs